__tests__/endpoints.test.js

```diff
@@ -103,11 +107,88 @@ describe('GET /api/articles/:article_id', () => {
         return request(app)
         .get('/api/articles/999')
         .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe('article does not exist');
+        });
     });
     test('should return a 400 status code when passed an invalid article number', () => {
         return request(app)
         .get('/api/articles/pies')
         .expect(400)
+        .then((response) => {
+            expect(response.body.msg).toBe('Bad request');
+        });
     });
 });
-    
\ No newline at end of file
+    
+describe.only('GET /api/articles', () => {
+    test('should return a 200 status code ', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+    });
+    test('should respond with articles with the following keys', () => {
+        return request(app)
+        .get('/api/articles')
+        .then((response) => {
+            const articles  = response.body.articles;
+            expect(articles.length).toBe(13)
+            articles.forEach((article) => {
+                expect(article).toHaveProperty("author")
+                expect(article).toHaveProperty("title")
+                expect(article).toHaveProperty("article_id")
+                expect(article).toHaveProperty("topic")
+                expect(article).toHaveProperty("created_at")
+                expect(article).toHaveProperty("votes")
+                expect(article).toHaveProperty("article_img_url")
+                expect(article).toHaveProperty("comment_count");
```

Comment:
This works fine here, but for future tests take a look into the jest matchers toMatchObject and expect.ObjectContaining, they just all for more robust yet flexible testing.